### ISSUE TITLE
Linux packet 3.4.0/3.5.0?

### ISSUE THREAD

**Issue by SergeyDjam** [meganz/MEGAsync#144]
I did not find a packet for Linux in the section of releases

**Comment by baizon** [comment]
Did you check: https://mega.nz/sync
Also there is the direct link: https://mega.nz/linux/MEGAsync/

**Comment by SergeyDjam** [comment]
https://github.com/meganz/MEGAsync/releases ?

**Comment by polmr** [comment]
@SergeyDjam, you are right, the tag is missing. We will add it. This is the commit that it should point to: https://github.com/meganz/MEGAsync/commit/f29c847b9b58407d9d78f9cbb58166ab514dad5e
